### DISCUSSION TITLE
No Youtube video: This video isn't available anymore

### DISCUSSION THREAD

**Discussion by NoRainfallDuckDog** [netblue30/firejail#6501] [General]
https://firejail.wordpress.com/2021/01/31/firejail-security-deep-dive-video/

https://www.youtube.com/watch?v=N-Mso2bSr3o

**Comment by rusty-snake** [comment]
@netblue30

**Reply by amano-kenji** [reply]
Youtube censorship comes to your door. I don't think youtube makes mistakes when they drop videos.
